src/openforms/js/components/admin/form_design/logic/actions/dmn/DMNActionConfig.js

```diff
@@ -185,7 +185,7 @@ const DMNActionConfig = ({initialValues, onSave}) => {
       >
         {formik => (
           <Form>
-            <fieldset className="aligned">
+            <fieldset className="admin-fieldset aligned">
```

Comment:
I'm curious whether using these components fixes the issue or not. Otherwise, I would compare the markup and styling from a non-JS enhanced admin page with validation error and see what the HTML/class names look like.

Since this is styling inside of a modal, I think we should be able to replicate the appearance without CSS overrides, but I'm not sure, there may be some top-level classes missing because the admin CSS is... not pleasant.